packages/web_benchmarks/lib/src/runner.dart

```diff
@@ -117,16 +116,16 @@ class BenchmarkServer {
         Completer<List<Map<String, dynamic>>>();
     final List<Map<String, dynamic>> collectedProfiles =
         <Map<String, dynamic>>[];
-    List<String> benchmarks;
-    Iterator<String> benchmarkIterator;
+    List<String>? benchmarks;
```

Comment:
Why is this nullable? We strongly prefer non-nullable collections to avoid having two ways of expressing empty, which is very error-prone to use.

Reply:
If it is null, we populate it [here](https://github.com/amanv8060/packages/blob/29bd0f0a383c87ca9684e049b90a5835b293c954/packages/web_benchmarks/lib/src/runner.dart#L192).